src/test/java/com/google/api/codegen/testdata/discoveries/go/go_storage.v1.json.baseline

```diff
@@ -149,14 +131,13 @@ func main() {
   }
 
   // Name of a bucket.
-  bucket := "{MY-BUCKET}"
-  // TODO: Change placeholder value above.
+  bucket := ""  // TODO: Update placeholder value.
 
-  rb := storage.BucketAccessControl{
+  rb := &storage.BucketAccessControl{
```

Comment:
I would like to leave this as is - I really don't think that where we take the reference of the object impacts readability or improves any highlighting. Specifically, no Go developer will be impacted.

Two reasons:
- it requires another line of logic in the template
- zero values for messages, should they ever happen to exist will be defined as the reference to a new object: &Empty{}